.changeset/tricky-singers-brush.md

```diff
@@ -0,0 +1 @@
+Updates v2 core CHANGELOG
```

Comment:
I dont think we need the changeset at all?

Reply:
Ok. I can delete. I was just following the contributing guide as I had before. But makes sense since there's no code or patch.